gazebo/transport/Node.hh

```diff
@@ -18,7 +18,9 @@
 #ifndef GAZEBO_TRANSPORT_NODE_HH_
 #define GAZEBO_TRANSPORT_NODE_HH_
 
+#undef emit
 #include <tbb/task.h>
+#define emit
```

Comment:
I was a bit afraid of doing so as it changes also the case tbb<2021. Do you think that defining `emit` as empty is ok?

Reply:
@traversaro ``emit`` is actually defined by Qt as empty too! It's just convention that it's used in Qt code as a pseudo-keyword.

Reply:
Ahh, cool.

Reply:
I hadn't thought clearly about the implications; I was just trying to get to a working build. We can wrap in `#ifdef`s if needed

Reply:
@scpeters Yes but that shouldn't be necessary. As I said, qt just defines it as an empty macro too.